### DISCUSSION TITLE
Reconfiguring HA cluster to use experimental tailscale integration

### DISCUSSION THREAD

**Discussion by ostapkostyk** [k3s-io/k3s#10026] [Q&A]
Hello,

I have a HA multicloud cluster configured with ```--node-external-ip=<SERVER_EXTERNAL_IP> --flannel-backend=wireguard-native --flannel-external-ip``` options.

Now I ran into an [issue with MTU](https://github.com/k3s-io/k3s/issues/9097) when trying to connect agent over tailscale vpn. The agent runs but I have issues with Longhorn when rebuilding data volumes, there are drops in communication.

So I'm thinking to re-configure the cluster to use experimental tailscale integration.   
But the cluster is already lets say in production, I spent a lot of time on it so I'm looking for the way to re-configure it instead of starting from scratch.

As I understood from the documentation, to use experimental integration, ```--vpn-auth="name=tailscale,joinKey=$AUTH-KEY"``` options should be used when deploying the cluster.  
Is there a way to switch from multi-cluster to tailscale integration for running HA cluster without re-installing it?

If not, shall the procedure be  
- backup ETCD
- re-install K3S from scratch using tailscale integration
- restore ETCD from backup

It looks risky for me, I'm not advanced k3s user. So I'm curious can just change settings for running cluster.

**Comment by cristianrat** [comment]
I would also like to know this
Might just have to restart a new cluster from snapshot
